resources/[core]/customization/src/client/managers/creation-manager.ts

```diff
@@ -54,6 +54,10 @@ class CreationManager {
 
   private currentGender: 'male' | 'female' = 'male';
 
+  private firstName = '';
```

Comment:
Should use _firstName for private. And below instead of `setFirstName` use the `set` keyword

```ts
set firstName(name: string) {
  this._firstName = name
}
```

Same for other privates and sets.

Reply:
changed for _ instead of private and removed the set functions